README.md

```diff
@@ -77,9 +77,6 @@ npm install && npm run update
 npm run dev
 ```
 
-### Is svelte.dev down?
```

Comment:
I'm not sure why this is showing in the diff. Perhaps you need to rebase against `master`

Reply:
this is embarrassing